inc/field/dropdownfield.class.php

```diff
@@ -470,7 +470,7 @@ public function isValidValue($value): bool {
    public function prepareQuestionInputForSave($input) {
       if (!isset($input['itemtype']) || empty($input['itemtype'])) {
          Session::addMessageAfterRedirect(
-            sprintf(__('The field value is required: %s', 'formcreator'), $input['name']),
+            sprintf(__('The itemtype field is required: %s', 'formcreator'), $input['name']),
```

Comment:
this message is wrong : the alue of a dropdown w as an itemtype, but a dedicated columns has been added in DB a few months ago